README.md

```diff
@@ -72,16 +72,16 @@ To use the CDN, include links to the JS and CSS files in your HTML:
 
 ```html
 <!-- Latest CDN production Javascript and CSS -->
-<script src="https://global.oktacdn.com/okta-signin-widget/3.8.2/js/okta-sign-in.min.js" type="text/javascript"></script>
+<script src="https://global.oktacdn.com/okta-signin-widget/3.9.1/js/okta-sign-in.min.js" type="text/javascript"></script>
```

Comment:
this takes a few days to be available right? But by changing it here it means if someone looks at this readme after you merge this to master, they won't find that version... might be better to update to 3.9.0? 
@nikhilvenkatraman-okta ?

Reply:
It will be published and available at this URL before I backport these changes to master. It must be available before I can add this version to okta-core. (It is done by the conductor workflow)

Reply:
talked offline, global cdn is updated right away